### ISSUE TITLE
More Constructions Without Factory not turning off correctly

### ISSUE THREAD

**Issue by beelee1** [triplea-game/triplea#1466]
Don't know if this is already known about. I have More Constructions without Factory set to false and editable to true. When I start triplea it is not turned off. If I leave the game without shutting off triplea, when I select the game again it will be turned off.

I also have More Constructions with factory set to true editable to true and Unlimited Constructions set to false editable true. They both behave correctly.

Not a huge problem but I hadn't noticed it before so I thought I'd mention it.

**Comment by ron-murhammer** [comment]
@beelee1 Can you provide a little more detail here, I'm not really following you. Can you provide specific steps to reproduce the issue (screenshots help as well)?

**Comment by beelee1** [comment]
I changed the global xml so that with/without factories and unlimited constructions were all set to editable = true. I also had with factories set to true with the other two set to false. 

When you first start up triplea without factories is on, which it shouldn't be. Start the game and then exit back to triplea, check map options again and it's off, which is correct.

Here's some shots
![more constructions without factory](https://cloud.githubusercontent.com/assets/24382682/22008141/f5b445ea-dc3d-11e6-98b8-a3d26d76275a.png)

![when starting triplea](https://cloud.githubusercontent.com/assets/24382682/22008144/06896bca-dc3e-11e6-9de9-283f4e9bd8e3.png)

![after starting game then exiting game](https://cloud.githubusercontent.com/assets/24382682/22008149/11459610-dc3e-11e6-81a4-8f2fcd4c9280.png)

**Comment by simon33-2** [comment]
I can't see the problem. Did you drop out of Triple-A after changing the .xml file? I wouldn't expect it to detect a change.

If I change the xml then start Triple-A the option is correctly detected.

**Comment by beelee1** [comment]
I just tried the change on a clean global xml ( no changes ) and it works correctly. Guess I should've double checked that first.

I don't understand why it's doing it on my mod though. Yea I had triplea turned off, made the change, fire it back up. It must not have liked something I did.

**Comment by simon33-2** [comment]
Perhaps there were two copies of the map?

**Comment by beelee1** [comment]
yea I checked that. I always make the change to the downloaded maps folder so I can just fire it up to see if it works, then when it does, I update the backup. So the only thing not working for me right now is this. Which isn't a huge deal because of the easy workaround.

It just seems as if it should be pretty basic. Unfortunately I've already made a pile of changes before this one, so backtracking could take a few days.
